easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-1.9.0-gompi-2017a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCoarrays'
+version = '1.9.0'
+
+homepage = 'https://github.com/sourceryinstitute/opencoarrays'
+description = """OpenCoarrays is an open-source software project that supports
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/sourceryinstitute/opencoarrays/releases/download/%(version)s']
+
+sources = [SOURCE_TAR_GZ]
+
+separate_build_dir = True
+
+builddependencies = [('CMake', '3.8.2')]
+
+sanity_check_paths = {
+    'files': ['bin/caf', 'bin/cafrun', 'include/opencoarrays.mod', 'lib/libcaf_mpi.a'],
```

Comment:
@anujsharmax `libcaf_mpi.a` is getting installed in `lib64/` rather than `lib` on my system (CentOS 7),
so let change this to:

```python
    'files': ['bin/caf', 'bin/cafrun', 'include/opencoarrays.mod', ('lib/libcaf_mpi.a', 'lib64/libcaf_mpi.a')],
```

the `('lib/...', 'lib64/...)` syntax specifies two alternatives for the location of `libcaf_mpi.a`; if one of either is found, the sanity check will pass